jdaviz/configs/default/plugins/viewers.py

```diff
@@ -316,6 +316,13 @@ def reference_id(self):
     def reference(self):
         return self.jdaviz_app._viewer_item_by_id(self.reference_id).get('reference')
 
+    @property
+    def _ref_or_id(self):
+        reference = self.reference
```

Comment:
this can be shortened (but also nbd if you want to keep it as is)
return self.reference_id if self.reference is None else self.reference

Reply:
You mean `self.reference_id if self.reference is not None else self.reference` ?

Reply:
I avoided that since `self.reference` isn't cached and requires looping over the viewer-store and I didn't want to have to call that twice.  Is it worth implementing caching so that we can do this?

Reply:
Naw.

Reply:
Might want to add a note about this so future dev won't try to shorten it?

Reply:
I didn't merge because I don't know if you want to follow up on this, so I'll let you merge when you think this is ready. You already have 2 approvals.